### PR TITLE
Make welcome page responsive

### PR DESCRIPTION
**Pull request by nachoiacovino**
All CSS changes that fix various bugs and improves the experience on mobile devices.

Before: 
image

After:
image

Please, add any comments on any thing that you think should be improved.

Thank you.

### CODE REVIEW

**Comment by colinclerk** [comment]
Looking much better, thanks!  Did full screen stay the same?

**Comment by nachoiacovino** [comment]
> Looking much better, thanks! Did full screen stay the same?

Oops, forgot to share. I did some slight changes to it too. 

Before:
image

After:
image

Tell me if you think it's good.

**Comment by nachoiacovino** [comment]
I push a couple more commits so it's now pixel perfect with Cooper's design.

image

image

File: styles/globals.css [hunk 1]
```diff
@@ -6,9 +6,9 @@ body {
   font-size: 16px;
   font-weight: 400;
   line-height: 20px;
-
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
```
Comment (nikosdouvlis) [inline styles/globals.css:11]: ❓ Hey @nachoiacovino! Would you mind explaining the reasoning behind this change? For this simple starter app, it mainly affects the distances between the card containers but if that's what you wanted to achieve I'd prefer keeping the global box-sizing rule and fixing the components locally

**Review by nikosdouvlis — COMMENTED** [review]
Very well done 🚀  Got a minor comment about `box-sizing`, but apart from that it looks good to me!

File: styles/globals.css (hunk 1 above)
Comment (nachoiacovino) [reply styles/globals.css:11]: It was breaking the html tag to have it global, if you check the "before" mobile screenshot, everything was pushed a little bit to the left, like having an invisible margin-right. This fixed it.

If you find another way of fixing it I would love to hear it 😁

**Review by colinclerk — APPROVED** [review]
LGTM! Thank you!

File: pages/index.js [hunk 2]
```diff
@@ -116,6 +116,10 @@ export default function Home() {
       <Head>
         <title>Create Next App</title>
         <link rel="icon" href="/favicon.ico" />
+        <meta
```
Comment (nikosdouvlis) [inline pages/index.js:119]: Good catch. Glad you figured this one out ✅

**Review by nikosdouvlis — APPROVED** [review]

**Comment by nachoiacovino** [comment]
@colinclerk catched it, credits to him 😁

**Pull request merged**